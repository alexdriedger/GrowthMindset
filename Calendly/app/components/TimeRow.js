import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';

import * as STYLES from '../common/Styles';

const styles = StyleSheet.create({
  container: {
    height: 120,
    backgroundColor: STYLES.COLOR_PRIMARY,
    justifyContent: 'center',
    alignItems: 'center',
  },
  text: {
    color: 'white',
    fontSize: 26,
    fontWeight: 'bold',
  },
});

class TimeRow extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.text}>{this.props.text}</Text>
      </View>
    );
  }
}

export default TimeRow;
