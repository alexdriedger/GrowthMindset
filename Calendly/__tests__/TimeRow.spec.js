import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TimeRow from '../app/components/TimeRow';

it('renders correctly', () => {
  const tree = renderer.create(<TimeRow text="lsb" />).toJSON();
  expect(tree).toMatchSnapshot();
});
