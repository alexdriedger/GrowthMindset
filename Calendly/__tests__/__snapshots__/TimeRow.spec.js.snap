// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderColor": "#00a3fa",
      "borderRadius": 20,
      "borderWidth": 5,
      "height": 80,
      "justifyContent": "center",
      "overflow": "hidden",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#222222",
        "fontSize": 28,
        "fontWeight": "bold",
      }
    }
  >
    lsb
  </Text>
</View>
`;
